protocol/x/clob/types/process_proposer_matches_events.go

```diff
@@ -0,0 +1,56 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+)
+
+// ValidateProcessProposerMatchesEvents performs basic stateless validation on ProcessProposerMatchesEvents.
+// It returns an error if:
+// - Any of the fields have duplicate OrderIds.
```

Comment:
Is it worth noting that this is currently invalid since stateful order replacements are not enabled?